articlesModel.js

```diff
@@ -0,0 +1,19 @@
+class ArticlesModel {
+  constructor() {
+    this.articles = [];
+  }
+
+  setArticles(articles) {
+    this.articles = articles;
+  }
+
+  getArticles() {
+    return this.articles;
+  }
+
+  reset() {
```

Comment:
The reset function might not be necessary, as the setArticles function overrides this.articles? I'm not entirely sure though lol I could be very wrong.